### DISCUSSION TITLE
1.6.0 final release

### DISCUSSION THREAD

**Discussion by joeltankam** [Excel-DNA/ExcelDna#483] [General]
Hello, 

Version `1.6.0` of the package has been in preview for a little while.
Is there any planned release date or period for the final release ?

Thanks.

**Comment by govert** [comment]
@joeltankam I have no fixed date in mind, but I think the progress is good.

The next preview will have some changes in the output directory for some files, and that will likely be the end the changes I wanted for this version - particularly with the changes on the build / project side. I'm also unlikely to make any changes to the runtime under .NET Framework before the final release, though we might try to fix what we can for the .NET 6.0 runtime.

What has been your experience with the 1.6.0 version so far?

**Comment by joeltankam** [comment]
Thanks for your reply. 

> What has been your experience with the 1.6.0 version so far?

`1.6.0-preview3` had some build issues in .NET Core last time I checked. But, `1.6.0-preview4` seems to work just fine.

**Comment by conor1833** [comment]
Hi is there an update on version 1.6?

**Comment by govert** [comment]
Version 1.6.0 has been available on NuGet for a few months. Do you mean you are looking for a standalone release?
The current pre-release version is 1.6.1-beta3.

**Reply by conor1833** [reply]
Ah that's great thanks! I was just wondering about 1.6.0  because I couldn't find a release notification. Is Nuget the best place to find out about releases and new features?
